packages/react-admin-stories/src/react-admin-core/StackPageTitle.tsx

```diff
@@ -0,0 +1,105 @@
+import { storiesOf } from "@storybook/react";
+import { IStackSwitchApi, Stack, StackPage, StackPageTitle, useStackSwitch } from "@vivid-planet/react-admin-core";
+import * as React from "react";
+import { Redirect, Route, Switch } from "react-router";
+import StoryRouter from "storybook-react-router";
+
+function Page2() {
+    const [counter, setCounter] = React.useState(0);
+    const { StackSwitch, api } = useStackSwitch();
```

Comment:
Isn't it that things can go wrong when a sombody uses the returned StackSwitch multiple times (Nested Stack). 
I think it's safe actually, but still the following is prefered when 2 Switches are needed:
```ts
    const { StackSwitch: StackSwitch1, api: api1 } = useStackSwitch();
    const { StackSwitch: StackSwitch2, api: api2 } = useStackSwitch();
```
Correct?

and... probably the api should be an array to avoid renaming in the deconstructor:

```ts
    const [ StackSwitch1, api1 ] = useStackSwitch();
    const [ StackSwitch2, api2 ] = useStackSwitch();

```

Reply:
changed to tuple